package.json

```diff
@@ -12,7 +12,7 @@
   "dependencies": {
     "@fortawesome/fontawesome-free": "^5.15.4",
     "@shoelace-style/shoelace": "~2.15.1",
-    "@webrecorder/wabac": "^2.18.3",
+    "@webrecorder/wabac": "github:webrecorder/wabac.js",
```

Comment:
Should we lock this down with a commit hash?

Reply:
oh, yes, this was just for testing! Will do a 2.18.4 release when all is ready.